sample_code_submission/statistical_analysis.py

```diff
@@ -245,15 +247,16 @@ def nominal_histograms(self, alpha, key):
         """
         syst_settings = self.syst_settings.copy()
         syst_settings[key] = alpha
-        holdout_syst = systematics(
-            self.holdout_set.copy(),
-            tes=syst_settings['tes'],
-            bkg_scale=syst_settings['bkg_scale'],
-            jes=syst_settings['jes'],
-            soft_met=syst_settings['soft_met'],
-            ttbar_scale=syst_settings['ttbar_scale'],
-            diboson_scale=syst_settings['diboson_scale'],
-        )
+        holdout_syst = self.holdout_set.copy()
+        # holdout_syst = systematics(
+        #     self.holdout_set.copy(),
+        #     tes=syst_settings['tes'],
+        #     bkg_scale=syst_settings['bkg_scale'],
+        #     jes=syst_settings['jes'],
+        #     soft_met=syst_settings['soft_met'],
+        #     ttbar_scale=syst_settings['ttbar_scale'],
+        #     diboson_scale=syst_settings['diboson_scale'],
+        # )
```

Comment:
Even if `systematics()` is not needed for stat only I would recommend using it but with all inputs set to default or not using those parameters. That would just do the job of post process cuts. 

But we should a comment there stating its done to have post process cuts. It would be simpler that a separate function. (`apply_postprocess_to_dict()`)

Reply:
Lets merge like this and the see if we need to change it. Because having a working model is more important than aesthetics

Reply:
In this case, I implement all the post-process cuts at the very beginning of `model.py` Line 78. So all the dataset processed in `model.py` will automatically have these cuts.